README.md

```diff
@@ -18,7 +18,7 @@ Agencies are encouraged to use the following HTML snippet to participate in the
 
 ```html
 <!-- We participate in the US government's analytics program. See the data at analytics.usa.gov. -->
-<script id="_fed_an_ua_tag" src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js"></script>
+<script async id="_fed_an_ua_tag" src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js"></script>
```

Comment:
Oh -- I also added in the `async` attribute here as part of the recommended snippet. That is unrelated to the rest of this issue, and I probably should have done it in another PR. If it merits further discussion, feel free to ask that I undo this before merge.